Source/WebInspectorUI/UserInterface/Views/OverrideUserPreferencesPopover.js

```diff
@@ -0,0 +1,150 @@
+
+        this._presentOverTargetElement();
+    }
+
+    dismiss()
+    {
+        this._defaultValueOptionElementsMap.clear();
+
+        super.dismiss();
+    }
+
+    // Private
+
+    _presentOverTargetElement()
+    {
+        if (!this._targetElement)
+            return;
+
+        let targetFrame = WI.Rect.rectFromClientRect(this._targetElement.getBoundingClientRect());
+        const preferredEdges = [WI.RectEdge.MAX_Y, WI.RectEdge.MAX_X];
+
+        this.present(targetFrame.pad(2), preferredEdges);
+    }
+
+    _createSelectElement(contentElement, label, preferenceName, values, defaultValue)
+    {
+        if (!preferenceName || !values.length)
+            return;
+
+        let labelElement = contentElement.appendChild(document.createElement("label"));
+        labelElement.textContent = label;
+        labelElement.setAttribute("for", preferenceName);
+
+        let selectElement = contentElement.appendChild(document.createElement("select"));
+        selectElement.setAttribute("name", preferenceName);
+        selectElement.setAttribute("id", preferenceName);
+
+        let systemValue = WI.cssManager.defaultUserPreferences.get(preferenceName);
+
+        for (let value of [defaultValue, ...values]) {
+            let optionElement = selectElement.appendChild(document.createElement("option"));
+            optionElement.value = value;
+            optionElement.textContent = this._userPreferenceValueLabel(value, systemValue);
+
+            let selectedValue = WI.cssManager.overridenUserPreferences.get(preferenceName) || defaultValue;
+            if (value === selectedValue)
+                optionElement.setAttribute("selected", true);
+
+            if (value === defaultValue)
+                this._defaultValueOptionElementsMap.set(preferenceName, optionElement);
+        }
+
+        selectElement.addEventListener("change", (event) => {
+            let value = event.target.value;
+            if (value !== defaultValue)
+                WI.cssManager.overrideUserPreference(preferenceName, value);
+            else
+                WI.cssManager.overrideUserPreference(preferenceName); // Pass no value to remove the override and restore the system default.
+        });
+    }
+
+    _createContentElement()
+    {
+        let contentElement = document.createElement("div");
+        contentElement.classList.add("user-preferences-content");
+
+        let accessibilityHeader = contentElement.appendChild(document.createElement("h1"));
+        accessibilityHeader.textContent = WI.UIString("Accessibility Preferences", "Accessibility Preferences @ User Preferences Overrides", "Header for section with accessibility user preferences.");
+
+        const userPreferenceValues = [InspectorBackend.Enum.Page.UserPreferenceValue.On, InspectorBackend.Enum.Page.UserPreferenceValue.Off];
+
+        // COMPATIBILITY (macOS 13.0, iOS 16.0): `PrefersReducedMotion` value for `Page.UserPreferenceName` did not exist yet.
```

Comment:
Since we should never reach here when inspecting a target running an older OS, I don't think we need these compatibility checks. But maybe we do want to assert in the constructor that `InspectorBackend.hasCommand("Page.overrideUserPreference")` is true, or a similar expression (with a compat comment as well). Then we can safely assume in this entire file that the stuff you are adding in the patch exists.

Reply:
I'll keep this as-is and not guard this whole component behind `InspectorBackend.hasCommand("Page.overrideUserPreference")` because I have a follow-up patch which implements the backwards-compatible method to toggle appearance with the same UI.